
public class ExpHandlDemo {

	public static void main(String[] args) throws Exception {
 
		try {
			int a[] =new int[5];
			int j=4;
			int i=2;
			int k= i/j;
 			//System.out.println(a[4]);
			 System.out.println(k);
			
			if (k==0)
				throw new TabException("User Tab caught the exception");

		}
		catch (TabException e) {
			System.err.println("Error!! " + e.getMessage());
		}
		finally {
			 System.out.println("In finally block");
		}

 	}

}
